sdk.ts

```diff
@@ -1226,6 +1229,35 @@ export class Client {
                     }),
             );
         },
+        /**
+         * @unstable
+         */
+        getOwnerForLocation: async (args: {
+            locationId: number;
+        }) => {
+            const accounts = await this.getAccountsForLocation({ locationId: args.locationId });
+            if (accounts instanceof Error) {
+                return accounts;
+            }
+            if (accounts.length > 1) {
+                console.error(`more than one owner for the location, locationId: ${args.locationId}`);
+            }
+            for (const account of accounts) {
+                const pubkey = PublicKey.FromHex(account.pubKey);
+                if (pubkey instanceof Error) {
+                    console.error(`account ${account.id} has invalid pubkey`, pubkey);
```

Comment:
this should never happen and if it happened, it's a backend bug.

Reply:
But if you don't write it this way, the compiler will report a type error. Should I return `InvalidKey` directly instead of skipping?